core/sample.h

```diff
@@ -23,7 +24,7 @@ struct sample                         // BASE TYPE BYTES  UNITS    RANGE
 	temperature_t temperature;        // uint32_t   4    mK     (0-4 MK)               ambient temperature
 	pressure_t pressure[MAX_SENSORS]; // int32_t  2x4    mbar   (0-2 Mbar)             cylinder pressures (main and CCR o2)
 	o2pressure_t setpoint;            // uint16_t   2    mbar   (0-65 bar)             O2 partial pressure (will be setpoint)
-	o2pressure_t o2sensor[3];         // uint16_t 3x2    mbar   (0-65 bar)             Up to 3 PO2 sensor values (rebreather)
+	o2pressure_t o2sensor[MAX_O2_SENSORS];// uint16_t 6x2    mbar   (0-65 bar)             Up to 6 PO2 sensor values (rebreather)
```

Comment:
I wonder at which point this should become a dynamic table dependent on `dc->no_o2sensors`.

Reply:
I would propose I do that the next time the number of sensors has to be increased. Already now, this is the special case (Divesoft Liberty) of a special case (CCR dives). Already now, I think four (two redundant units that are in themselves already redundant) is a bit high...